dev/integration_tests/channels/android/app/build.gradle

```diff
@@ -37,11 +37,7 @@ android {
     }
 
     aaptOptions {
-        // TODO(goderbauer): remove when https://github.com/flutter/flutter/issues/8986 is resolved.
-        if(System.getenv("FLUTTER_CI_WIN")) {
-            println "AAPT cruncher disabled when running on Win CI."
-            cruncherEnabled false
-        }
+        
     }
```

Comment:
can we remove the entire aaptOptions section? (I'm not sure, I'm not a gradle expert.)

Reply:
Do you have to ask anyone for this @Hixie?

Reply:
I was asking you :-)
Maybe someone in #hackers knows.

Reply:
Hello @Hixie 
can you please ask any member of your team?
Thanks

Reply:
This can be removed as far as I know.  This will also pass the pipeline because "white spaced line" was causing it.